pkg/operator/ceph/object/notification/controller.go

```diff
@@ -195,10 +214,13 @@ func (r *ReconcileNotifications) reconcile(request reconcile.Request) (reconcile
 			notification,
 		)
 		if err != nil {
+			r.recorder.Eventf(notification, kapiv1.EventTypeNormal, string(cephv1.ReconcileFailed), "Reconcile failed for %s with error %s", bnName, err.Error())
 			return reconcile.Result{}, errors.Wrapf(err, "failed to provision CephBucketNotification %q for ObjectBucketClaims %q", bnName, bucketName)
 		}
+		r.recorder.Eventf(notification, kapiv1.EventTypeNormal, string(cephv1.Provisioned), "Notification %q provisioned on bucket %q", bnName, bucketName)
```

Comment:
I thought we had agreed that adding "Provisioned" events was going to be too much and that notifications should be later put into the status.

Reply:
ok. will remove